receiver/googlecloudpubsubreceiver/README.md

```diff
@@ -33,21 +37,32 @@ receivers:
     encoding: raw_json
 ```
 
+The full list of settings exposed for this receiver are documented [here](./config.go).
+
+## Modes
+
+The receiver supports two modes: ' push' and `pull`. If you don't explicitly set the mode in the config, the default is `pull`.
```

Comment:
```suggestion
The receiver supports two modes: `push` and `pull`. If you don't explicitly set the mode in the config, the default is `pull`.
```

Reply:
I've pushed it along with my single fixup commit.